tests/server_tests.py

```diff
@@ -164,10 +165,18 @@ def flush_output(self):
       """Flushes the buffered output from this subprocess to stderr."""
       self.output, original_output = [], self.output
       if original_output:
-          matchOB = self.BIND_RE.search('\n'.join(original_output), re.MULTILINE);
-          if matchOB:
-              sys.stderr.write('%s failed %s port %s is already in use.\n\n' %
-                               (self.name, matchOB.group(1), matchOB.group(2)))
+          error = '\n'.join(original_output)
```

Comment:
Calling this as "error" is a bit weird because it's an "output", not necessarily an error message. (It can be just some logs, I guess.) How about "original_output_text"?